### PR TITLE
Saransh sharma

### PR DESCRIPTION
**Pull request by Saransh1Sharma**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,4 +27,5 @@ Lines starting with `*` are automatically rendered as bullet points and phrases
 * Harsh Gupta [hargup](http://github.com/hargup)
 * Krishna Bagadia [krishna95](http://github.com/krishna95)
 * Nevin Valsaraj [routeaccess](http://github.com/routeaccess)
+* Saransh Sharma [sharma.saransh](http://github.com/sharma.saransh)
```
Comment (Saransh1Sharma) [inline README.md]: I have added my name.

**Comment by hargup** [comment]
There are merge conficts, you need to fix them.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 This is a place to practice git. Your first exercise is to add your name to this list.
-To complete this exercise you need to:
+To complete this exercise ou need to:
 
-- **Learn to use git**:
+- **Learn to use git in a comprehensive manner**:
```
Comment (hargup) [inline README.md]: These changes are related to adding your name in the list. If you want to suggest changes to the guidelines, then open a separate pull request.

**Comment by shivamvats** [comment]
Closed in favour of #71

File: README.md (hunk 2 above)
Comment (Saransh1Sharma) [inline README.md]: No suggestions. I was just experimenting.

**Pull request closed without merging**